src/main/java/cn/nukkit/level/chunk/ChunkSection.java

```diff
@@ -61,7 +61,7 @@ void checkLayer(int layer) {
     public Block getBlock(int x, int y, int z, int layer) {
         checkBounds(x, y, z);
         checkLayer(layer);
-        return this.storage[layer].getBlock(blockIndex(x, y, z)).clone();
```

Comment:
This is required otherwise blocks in the registry could be changed causing the palette to be invalid.

Reply:
the storage class also returns a .clone(), so do you really need a .clone() of a .clone() ?

Reply:
All good then 👍